src/main/common/color.h

```diff
@@ -35,20 +37,32 @@ typedef union {
     uint8_t raw[RGB_COLOR_COMPONENT_COUNT];
 } rgbColor24bpp_t;
 
+typedef union {
+    struct {
+        uint8_t r;
+        uint8_t g;
+        uint8_t b;
+        uint8_t w;
+    } rgbw;
+    uint8_t raw[RGBW_COLOR_COMPONENT_COUNT];
+} rgbwColor32bpp_t;
+
 #define HSV_HUE_MAX 359
 #define HSV_SATURATION_MAX 255
 #define HSV_VALUE_MAX 255
+#define HSV_WHITE_MAX 255
 
 typedef enum {
     HSV_HUE = 0,
     HSV_SATURATION,
-    HSV_VALUE
+    HSV_VALUE,
+    HSV_WHITE
 } hsvColorComponent_e;
 
-#define HSV_COLOR_COMPONENT_COUNT (HSV_VALUE + 1)
-
+#define HSV_COLOR_COMPONENT_COUNT (HSV_WHITE + 1)
 typedef struct hsvColor_s {
     uint16_t h; // 0 - 359
     uint8_t s; // 0 - 255
     uint8_t v; // 0 - 255
+    uint8_t w; // 0 - 255
```

Comment:
this doesn't make any sense here as Hue Saturation and Value can already be used to express White.